### PR TITLE
Documentation for AtlasTexture.

### PR DESCRIPTION
**Pull request by StraToN**

### CODE REVIEW

**Comment by NathanLovato** [comment]
Please check the [writing guidelines](https://github.com/GDquest/godot-3-guides/blob/master/contribution-guides/class-reference-styleguide.md), there's a lot of of content you can trim in the descriptions, e.g.:
```
Texture resource, basically a big texture aimed at allowing multiple, small textures to be contained in a single image file.
```
->
```
Packs multiple small textures in a single, bigger one. Helps to optimize video memory costs and render calls.
```

File: doc/base/classes.xml [hunk 1]
```diff
@@ -49422,7 +49429,7 @@
 		 - Rounded corners (individual radius for each corner)
 		 - Shadow
 		 About corner radius:
-		 	Setting corner radius to high values is allowed. As soon as corners would overlap the stylebox will switch to a relative system. Example: 
+		 	Setting corner radius to high values is allowed. As soon as corners would overlap the stylebox will switch to a relative system. Example:
```
Comment (NathanLovato) [inline doc/base/classes.xml]: Avoid passive voice:
```Setting corner radius to high values is allowed.```
-> direct voice is always shorter, and a lot easier to understand for non native and young speakers
```You may use high values for 'corner radius'.```

File: doc/base/classes.xml [hunk 2]
```diff
@@ -6052,10 +6055,14 @@
...
 	<members>
 		<member name="atlas" type="Texture" setter="set_atlas" getter="get_atlas" brief="">
+			The texture that contains the atlas. Can be any Texture subtype.
 		</member>
 		<member name="margin" type="Rect2" setter="set_margin" getter="get_margin" brief="">
+			The margin around the region. 'x' and 'y' respectively define left and top gaps that margin adds to the resulting region. 'w' and 'h' respectively
+			define right and bottom margins. The two latter resize the texture so it fits in the defined margin.
 		</member>
 		<member name="region" type="Rect2" setter="set_region" getter="get_region" brief="">
+			The region of the AtlasTexture that is actually used.
```
Comment (NathanLovato) [inline doc/base/classes.xml]: Abuse 's:
```The AtlasTexture's active/used region.```

File: doc/base/classes.xml [hunk 3]
```diff
@@ -6052,10 +6055,14 @@
 	</methods>
 	<members>
 		<member name="atlas" type="Texture" setter="set_atlas" getter="get_atlas" brief="">
+			The texture that contains the atlas. Can be any Texture subtype.
 		</member>
 		<member name="margin" type="Rect2" setter="set_margin" getter="get_margin" brief="">
+			The margin around the region. 'x' and 'y' respectively define left and top gaps that margin adds to the resulting region. 'w' and 'h' respectively
+			define right and bottom margins. The two latter resize the texture so it fits in the defined margin.
```
Comment (NathanLovato) [inline doc/base/classes.xml]: Favor explicit references over implicit ones. "latter" isn't a very common English word:
```The two latter resize the texture so it fits in the defined margin.```
-> also remove the passive voice:
```
The Rect2's 'w' and 'h' resize the texture so it fits within the margin.
```

File: doc/base/classes.xml [hunk 4]
```diff
@@ -6003,8 +6003,11 @@
 </class>
 <class name="AtlasTexture" inherits="Texture" category="Core">
 	<brief_description>
+		Texture resource, basically a big texture aimed at allowing multiple, small textures to be contained in a single image file.
 	</brief_description>
 	<description>
+		Texture resource aimed at managing big textures files that pack multiple smaller textures. Consists in a Texture, a margin that defines the border width, and a Region
```
Comment (Maxeonyx) [inline doc/base/classes.xml]: 'Consists of', rather than 'in'

**Comment by StraToN** [comment]
@NathanLovato Fixed.

File: doc/base/classes.xml [hunk 5]
```diff
@@ -49421,8 +49427,7 @@
 		 - Border width (individual width for each border)
 		 - Rounded corners (individual radius for each corner)
 		 - Shadow
-		 About corner radius:
-		 	Setting corner radius to high values is allowed. As soon as corners would overlap the stylebox will switch to a relative system. Example: 
+		 You may use high values for 'corner radius'. As soon as corners would overlap the stylebox will switch to a relative system. Example:
 			[codeblock]
```
Comment (Noshyaar) [inline doc/base/classes.xml]: I think codeblocks should be in the same indentation level as paragraphs?

**Comment by akien-mga** [comment]
There are now merge conflicts.

**Comment by StraToN** [comment]
@akien-mga Fixed :)
@Noshyaar Fixed :)

**Comment by akien-mga** [comment]
This will need to be redone as it was lost when the doc XML splitting was committed: 4f929a0

**Comment by StraToN** [comment]
@akien-mga No problem.

**Pull request merged**